lms/envs/devstack.py

```diff
@@ -27,7 +27,7 @@
 
 ################################ EMAIL ########################################
 
-EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+#EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
Commented out, but will be sticking around?  I'd just remove this line if the plan is to keep it commented forever.  If we're keeping it for a specific purpose, then I'd add a comment describing the use case in which to uncomment.